README.md

```diff
@@ -26,6 +26,26 @@ again, and was frustrutated when I couldn't find a similar plugin for vim, so
 after reading Steve Losh's 'Learn Vimscript the Hard Way' I decided to write one
 for myself.
 
+### Installation
+
+Using vim >=8.0, the builtin *packages* system is recommended:
```

Comment:
I don't think I would recommend this. Many people use vundle, pathogen, etc. to handle this stuff in a more automatic way.